### DISCUSSION TITLE
Visit do domain not registered

### DISCUSSION THREAD

**Discussion by dj-dule** [plausible/analytics#2920] [Anything Else]
Hi, I am using paid version of site. I installed script in header as requested. All visits to https://www.selectedgreen.com are registered, but visits to https://selectedgreen.com not. Do you know why and how I can fix this ? Thanks

**Comment by metmarkosaric** [comment]
thanks for reaching out @dj-dule! tracking seems to be working all fine on both sites. note that the stats will be showing on the same dashboard for both sites

**Reply by dj-dule** [reply]
Hmm, it seems yes, but I still do not see any visitors from Serbia (tried directly from within country and through VPN). Are data that I see through dashboard real time ?

**Reply by metmarkosaric** [reply]
Yes, it's all in real time. We have a post here on how you can test the integration: https://plausible.io/docs/troubleshoot-integration
